src/phoenix/datasets/schema.py

```diff
@@ -34,29 +34,20 @@ class Schema(Dict[SchemaFieldName, SchemaFieldValue]):
     prediction_score_column_name: Optional[str] = None
     actual_label_column_name: Optional[str] = None
     actual_score_column_name: Optional[str] = None
+    prompt_column_names: Optional[EmbeddingColumnNames] = None
+    response_column_names: Optional[EmbeddingColumnNames] = None
     embedding_feature_column_names: Optional[EmbeddingFeatures] = None
     excluded_column_names: Optional[List[str]] = None
 
-    def to_json(self) -> str:
-        "Converts the schema to a dict for JSON serialization"
-        dictionary = {}
+    def replace(self, **changes):
+        return replace(self, **changes)
```

Comment:
Add method to replace schema fields. Usability studies have revealed users are not familiar with `dataclasses.replace` function. It is beneficial to make it a method of our class